service/signinglog/actions.go

```diff
@@ -37,6 +37,7 @@ type ListResponse struct {
 	ErrorSubcode string                 `json:"error_subcode"`
 	ErrorMessage string                 `json:"message"`
 	SigningLog   []datastore.SigningLog `json:"logs"`
```

Comment:
afaiu these will containt total_count as well? that's a bit strange, that could be made at least not JSON serialized in SigningLog itself

Reply:
yes, this is correct, I just wanted to have the counter on the top level of the data